examples/tutorials/analysis-time/light_curve_flare.py

```diff
@@ -301,21 +304,37 @@
 # Finally we plot the result for the 1D lightcurve:
 #
 plt.figure(figsize=(8, 6))
-lc_1d.plot(marker="o")
+plt.tight_layout()
+plt.subplots_adjust(bottom=0.3)
+lc_1d.plot(marker="o", axis_name="time", sed_type="flux")
 plt.show()
 
 
 ######################################################################
-# Light curves once obtained can be rebinned.
-# Here, we rebin 4 adjacent bins together, to get 30 min bins
+# Light curves once obtained can be rebinned using the likelihood profiles.
+# Here, we rebin 3 adjacent bins together, to get 30 minute bins.
+#
+# We will first slice `lc_1d` to obtain the lightcurve in the first energy bin
 #
 
-axis_new = get_rebinned_axis(lc_1d, method="fixed-bins", group_size=3, axis_name="time")
+slices = {"energy": slice(0, 1)}
```

Comment:
This could be shown in the estimators notebook instead, no?

Reply:
we can have it repeated here